src/handler.py

```diff
@@ -31,13 +32,27 @@
     "QA": ['aqts-capture-trigger-QA-aqtsCaptureTrigger'],
     "PROD": ['aqts-capture-trigger-PROD-EXTERNAL-aqtsCaptureTrigger']
 }
+
+STAGE = os.getenv('STAGE', 'TEST')
+CAPTURE_TRIGGER_QUEUE = f"aqts-capture-trigger-queue-{STAGE}"
+ERROR_QUEUE = f"aqts-capture-error-queue-{STAGE}"
+
+DEFAULT_DB_CLUSTER_IDENTIFIER = f"nwcapture-{STAGE.lower()}"
+DEFAULT_DB_INSTANCE_IDENTIFIER = f"{DEFAULT_DB_CLUSTER_IDENTIFIER}-instance1"
+DEFAULT_DB_INSTANCE_CLASS = 'db.r5.8xlarge'
+ENGINE = 'aurora-postgresql'
+NWCAPTURE_REAL = f"NWCAPTURE-DB-{STAGE}"
+
 log_level = os.getenv('LOG_LEVEL', logging.ERROR)
 logger = logging.getLogger(__name__)
 logger.setLevel(log_level)
 
 STAGE = os.getenv('STAGE')
 
 cloudwatch_client = boto3.client('cloudwatch', os.getenv('AWS_DEPLOYMENT_REGION', 'us-west-2'))
+secrets_client = boto3.client('secretsmanager', os.getenv('AWS_DEPLOYMENT_REGION', 'us-west-2'))
```

Comment:
I wonder if it would be more effective to create a session object first. At least then the region wouldn't need to be passed in repeatedly.